.circleci/config.yml

```diff
@@ -70,12 +70,17 @@ workflows:
       - node/check_py3_compat_job
       - node/unittests_job:
           context: 
-            - slack-secrets
+            - plugins-inputs
           <<: *unittest-post-steps
       - wagonorb/wagon:
           filters:
             branches:
               only: /([0-9\.]*\-build|master|dev)/
+      - wagonorb/wagon_311
+      # - wagonorb/wagon_311:
+      #     filters:
+      #       branches:
+      #         only: /([0-9\.]*\-build|master|dev)/
```

Comment:
Do we want to build 311 wagon on all branches? Or is this filter somewhere else now?